apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue

```diff
@@ -144,6 +144,18 @@ const state = reactive({
                     ]).rawQueryStrings,
                 },
             };
+
+            const matchedJobIndex = collectorPageState.collectorJobStatus.findIndex((status) => status.collector_id === d.collector_id);
+            const recentJobAnalyze = collectorPageState.collectorJobStatus[matchedJobIndex]?.job_status.slice(-5) || [];
+
+            while (recentJobAnalyze.length < 5) {
+                const noneValue = {
+                    job_id: '',
+                    status: JOB_STATE.NONE,
+                };
+                recentJobAnalyze.unshift(noneValue);
```

Comment:
question: Type Inference occurs in this case?

Reply:
Although the type is specified in `CollectorItemInfo.vue`, which receives `state.item` as `props`, I will also specify it here!